pkg/apis/core/types.go

```diff
@@ -482,6 +482,15 @@ type PersistentVolumeClaimSpec struct {
 	// (Alpha) Using the namespace field of dataSourceRef requires the CrossNamespaceVolumeDataSource feature gate to be enabled.
 	// +optional
 	DataSourceRef *TypedObjectReference
+	// volumeAttributesClassName is the name of the VolumeAttributesClass required by the claim.
+	// If specified, the provisioner will create or update the volume with the attributes defined
+	// in the corresponding VolumeAttributesClass. Different with storageClassName, it can be
+	// changed after the claim is created but an empty string value is disallowed.
+	// More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#volumeattributesclass
+	// (Alpha) Using this field requires the VolumeAttributesClass feature gate to be enabled.
+	// +featureGate=VolumeAttributesClass
+	// +optional
+	VolumeAttributesClassName *string
```

Comment:
@sunnylovestiramisu If the VolumeAttributesClass is changed from true to false, should the `volumeAttributeClassName` field be droped from the Spec and Status?

Reply:
Why VolumeAttributesClass will change from true to false? It is not a bool?

Reply:
Sorry for the unclear expression. If the [VolumeAttributesClass feature-gate](https://github.com/carlory/kubernetes/blob/kep-3751/pkg/features/kube_features.go#L811) is changed from true to false, should the volumeAttributeClassName field be droped from the Spec and Status?